packages/ai-editor/src/browser/ai-editor-command-contribution.ts

```diff
@@ -175,3 +186,11 @@ export class AiEditorCommandContribution implements CommandContribution, MenuCon
         });
     }
 }
+
+function containsPrompt(args: unknown[]): args is (unknown | { prompt: string })[] {
+    return args.filter(arg => isPromptArg(arg)).length > 0;
```

Comment:
We can avoid looking through every element with `.some()`

```suggestion
    return args.some(arg => isPromptArg(arg));
```